.circleci/config.yml

```diff
@@ -703,12 +711,14 @@ jobs:
 
   run-test-ios-13:
     <<: *base-job
-    # M1 unsupported
-    resource_class: macos.x86.medium.gen2
     steps:
       - checkout
       - install-dependencies:
+          install_xcbeautify: false
+          install_mint: true
           install_swiftlint: << parameters.install_swiftlint >>
+      - macos/install-rosetta
```

Comment:
Oh it's part of the orb sweet

Reply:
Oh sorry, missed that! I think that would make sense if we executed that in fast lane, but as long as we are executing this in circleci, I think this is fine IMO